src/GameKit/GameKit.cs

```diff
@@ -21,19 +22,31 @@ namespace GameKit {
 #if !MONOMAC
 
 	// NSUInteger -> GKPeerPickerController.h
-#if XAMCORE_4_0
-	[NoTV] // preserve binary compatibility with existing/shipping code
```

Comment:
Manually removing this XAMCORE_4_0 and [NoTV]
and throwing in the [UnsupportedOSPlatform ("tvos")] into the `#if NET`